lore/end_users/status/README.md

```diff
@@ -40,3 +46,7 @@ The Zuul Pipelines counter provides an easy look at how many pipelines exist in
 ### Service Status
 
 Nodepool, Zookeeper, and Zuul are three core services that are essential to the stability of a BonnyCI cluster. If anything except the green "OK" message is displayed for these service statuses, then BonnyCI could be experiencing a full service outage.
+
+## Build Status
+
+The [Build Status page](https://travis-ci.org/BonnyCI) links to the Travis-CI page with our repositories. Yes we see the humor in a CI tool using a different CI tool for testing. This is a temporary home for us.
```

Comment:
@jesusaurus: I can make the subsections, that sounds like a happy compromise.

@jesusaurus and @j2sol: I am looking for a general build health page (for example: https://travis-ci.org/BonnyCI), but it doesn't necessarily need to have anything in depth. To the best of my knowledge there is nothing yet in BonnyCI that gives us that?